rs/https_outcalls/adapter/src/metrics.rs

```diff
@@ -38,6 +46,23 @@ impl AdapterMetrics {
                 "requests_socks_total",
                 "Total number of requests served via a SOCKS proxy",
             ),
+            socks_connections_attempts: metrics_registry.int_counter(
+                "socks_connections_attempts",
```

Comment:
counters usually as suffixed with _total at the end of the metric 
https://prometheus.io/docs/practices/naming/

Reply:
done